src/locales/pl.json

```diff
@@ -0,0 +1,27 @@
+{
+  "translation": {
+    "appName": "Jaka To Melodia",
+    "title": "🎵 $t(appName)",
+    "winMsg": "Wygrałeś!",
+    "guessPlaceholder": "Zgadnij tytuł",
+    "playXSeconds": "Odtwórz {{count}}s",
+    "guessBtn": "Zgadnij",
+    "skipBtn": "Pomiń",
+    "giveUp": "Poddaj się",
+    "nextSong": "Następna piosenka",
+    "menuEntry": "Zagraj w $t(appName)",
+    "sendingURIs_one": "Wysyłanie {{count}} URI do $t(appName)",
+    "sendingURIs_other": "Wysyłanie {{count}} URIs do $t(appName)",
+    "stats": {
+      "title": "Statystyki",
+      "time": "Czas",
+      "songs": "Piosenki",
+      "winPercentage": "Procent wygranych: {{percentage}}%",
+      "songWithCount_one": "{{count}} piosenka",
+      "songWithCount_other": "{{count}} piosenek",
```

Comment:
This spot needs additional keys added for Polish. See: https://www.i18next.com/translation-function/plurals#how-to-find-the-correct-plural-suffix

Reply:
Yeah, it did the trick <3
![Spotify_dfpuEKErTy](https://github.com/theRealPadster/name-that-tune/assets/3491056/e1ea402e-0426-4040-8865-d89f1d2f714d)